appveyor.yml

```diff
@@ -28,3 +28,19 @@ after_build:
 
 build_script:
 - cmd: build-vs2017.cmd
+
+artifacts:
+- path: build\*.nupkg
+  name: core
+
+deploy:
+- provider: NuGet
+  api_key:
+    secure: Please insert key here
```

Comment:
>I'm not sure what I'm meant to be checking out, are you politely telling me we should just use secure variables and be done with it even though they are sort of ugly? If so, I'll give in otherwise it seems I'm fighting against the grain of how AppVeyor wants to work even though secrets don't belong in source control.

No probs @jonorossi, just testing to water to get your thoughts. I did not think it was too bad but your way is definitely more secure. So let's try it. We can happily use environment variables. The build configuration for the project supports this under the environment link. 

![appveyor-env-var](https://cloud.githubusercontent.com/assets/607402/26194903/dcd6ac24-3bb1-11e7-93df-6d85ec76d695.png)

>How do we test this before going live against nuget.org, since you can't delete packages just unlisted them? MyGet?

The way I was approaching this, was to make changes to the `appveyor.yml` in fortress, and then see the tag publishing to NuGet followed by de-listing it. I am also using much lower version numbers so that it does not accidentally get in the way of an upgrade path. 

You can see the releases I have already tested with here(and will get rid of once this is complete): https://github.com/cryosharp/fortress/releases

Once I have fully validated the appveyor.yml, I was going give you instructions on how to setup the NuGet API key environment variable and then it should be a case of merging and hopefully presto! It just works.